core/src/main/java/nl/nn/adapterframework/filesystem/AmazonS3FileSystem.java

```diff
@@ -110,7 +114,8 @@ public void open() {
 		AmazonS3ClientBuilder s3ClientBuilder = AmazonS3ClientBuilder.standard()
 				.withChunkedEncodingDisabled(isChunkedEncodingDisabled())
 				.withForceGlobalBucketAccessEnabled(isForceGlobalBucketAccessEnabled()).withRegion(getClientRegion())
-				.withCredentials(new AWSStaticCredentialsProvider(awsCreds));
+				.withCredentials(new AWSStaticCredentialsProvider(awsCreds))
+				.withClientConfiguration(this.getProxyConfig());
```

Comment:
I can imagine that you should only add the client configuration if the proxy configuration has really been set,
and that if no proxy is configured, it should not be added.

Reply:
if getProxyConfig is null, the connection will be without proxy. we are not going to get a nullPointerException o something like that.